layouts/try/baseof.html

```diff
@@ -49,8 +49,28 @@ <h2>Interactive Tutorials</h1>
                 <div class="row">
                     <div class="col-md-4 pl-1 pr-1 pb-3">
                         <article class="card card-custom-style interactive-tutorial-trigger"
-                            style="background-color: hsl(171, 100%, 41%)" data-katacoda-src="ncskier/tekton-dashboard"
-                            data-github-lnk="ncskier/katacoda/tree/master/tekton-dashboard" data-qwiklabs-lnk="">
+                            style="background-color: hsl(171, 100%, 41%)" data-katacoda-src="ratrosyu/playground"
```

Comment:
We should add any additional tutorials into the website repo now that it's hooked up to the official Tekton Katacoda profile. Looks like this was one of the original examples by @michaelawyu and needs updating.

Since it's running kubernetes 1.14 some of the pods fail to start (min required kubernetes 1.15) and it's using a very old `tkn version`.

I have a [playground](https://katacoda.com/alangreene/scenarios/tekton-playground-minikube) with Pipelines 0.13.2 and Dashboard 0.7.0 on minikube that I was planning to contribute, and another in progress with a 2 node kubernetes 1.18 cluster that I could update to include `tkn` etc. as in Michael's original example.